data/sql/updates/pending_db_world/rev_1602390910121438300.sql

```diff
@@ -0,0 +1,3 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1602390910121438300');
+
+UPDATE `broadcast_text` SET `MaleText`='Have mercy, $gsir:madam;!', `FemaleText`='Have mercy, $gsir:madam;!' WHERE `ID`=28767;
```

Comment:
why

Reply:
Actually, I thought it was a mistake. I'm used to writing the **$g** after the word, so I made that modification, but I still have time to reverse it, because it's a WIP.

Reply:
@pangolp  has this been already addressed?

Reply:
In reality, change is not 100% necessary. It was my mistake, but it doesn't influence the final result. We can reverse it at last. Or even remove the line.

Reply:
There, I just removed the line. @Kitzunu @Si1ker